index.bs

```diff
@@ -252,7 +252,7 @@ The <dfn method for="StorageBucketManager">keys()</dfn> method steps are:
 
     1. If |bucket| is non-null, [=list/append=] |key| to |keys|.
 
-1. [=/Resolve=] |p| with |keys|.
+1. [=Queue a task=] to [=/resolve=] |p| with |keys|.
```

Comment:
Maybe add a note somewhere in the document that browsers will probably have IPC and disk accesses that make operations asynchronous.  Promises are resolved from tasks to permit this kind of work.